utils/create_cluster/local_cluster.py

```diff
@@ -80,17 +80,20 @@ def create_cluster(n_nodes, port):
         pids.append(pid)
     sleep(2)
     # Create cluster
-    redis_cli = os.getenv('REDIS_INSTALL_PATH') + '/redis-cli'
-    cluster_str=' '
-    for i in range(n_nodes):
-        cluster_str += '127.0.0.1:' + str(port+i) + ' '
-    cmd = " ".join((redis_cli, "--cluster create", cluster_str, "--cluster-replicas 0"))
-    print(cmd)
-    proc = run([cmd], input="yes", encoding="utf-8", shell=True)
-    if proc.returncode != 0:
-        raise SubprocessError("Cluster could not be created!")
+    if n_nodes > 1:
+        redis_cli = os.getenv('REDIS_INSTALL_PATH') + '/redis-cli'
+        cluster_str=' '
+        for i in range(n_nodes):
+            cluster_str += '127.0.0.1:' + str(port+i) + ' '
+        cmd = " ".join((redis_cli, "--cluster create", cluster_str, "--cluster-replicas 0"))
+        print(cmd)
+        proc = run([cmd], input="yes", encoding="utf-8", shell=True)
```

Comment:
Tangential to this pr, but
```py
proc = run([cmd], input="yes", encoding="utf-8", shell=True)
         # ^^^^^ Little thrown off by the list of length 1 with str with spaces
         #       I'm guessing this is due to `shell=True` weirdness
         #       Can we change to `cmd.split()` so that each symbol is its own element
         #       in the list just to conform to the `subprocess.run` docs?
```

Reply:
We actually fixed this when we migrated to the new launch_redis.py script. This file is no longer in the project as of the test system rewrite